rules/acv-terminaux.publicodes

```diff
@@ -0,0 +1,325 @@
+acv:
+  titre: Analyse de cycle de vie des appareils
+  description: |
+    Cette section présente les émissions de gaz à effet de serre liées à la fabrication, la distribution, l'usage et la fin de vie des appareils électroniques selon l'étude menée par Negaoctet.
+
+acv . smartphone:
+  description: |
+    6,63 inches screen mix of LCD and OLED technologies, 8 GB RAM, 210 GB memory (mix pondéré entre 3 configurations, 24%,24%, 52%)
+acv . smartphone . fabrication:
+  formule: 83000
+  unité: gCO2e
+acv . smartphone . distribution:
+  formule: 2100
+  unité: gCO2e
+acv . smartphone . usage julia:
+  formule: 600
+  unité: gCO2e
+acv . smartphone . usage théorique:
+  formule: consommation annuelle * publicodes-commun . intensité électricité * 1000 gCO2e/kgCO2e * durée de vie théorique
```

Comment:
qu'est-ce que `1000 gCO2e/kgCO2e` ici ?

Reply:
C'est pour la conversion et pour l'homogénéité des unités